src/lib.rs

```diff
@@ -61,6 +61,7 @@
 #![deny(missing_docs)]
 #![cfg_attr(all(test, windows), feature(std_misc))]
 
+#[allow(unused_imports)]
```

Comment:
I thought I would throw this in as well.

Reply:
What imports cause these warning?

Reply:
it only affects the next statement, so `use std::ascii::AsciiExt`. This was [introduced in recent rust versions](https://users.rust-lang.org/t/psa-dealing-with-warning-unused-import-std-ascii-asciiext-in-today-s-nightly/13726)

Reply:
Ah, yeah that makes sense then. :)